bundles/org.eclipse.equinox.p2.core/src/org/eclipse/equinox/p2/core/spi/AgentServiceName.java

```diff
@@ -23,14 +23,14 @@
 /**
  * This component property type can be used to annotate a declarative service
  * component that provides an {@link IAgentServiceFactory} to provides the
- * required {@link IAgentServiceFactory#PROP_CREATED_SERVICE_NAME}.
+ * required {@link IAgentServiceFactory#PROP_AGENT_SERVICE_NAME}.
  *
  * @since 2.13
  */
 @Retention(CLASS)
 @Target(TYPE)
 @ComponentPropertyType
-public @interface AgentServicename {
+public @interface AgentServiceName {
```

Comment:
I think the change is reasonable, I just wanted to mention that renamings that just change the case of some letters in a file name cause funny side-effects on Windows because the file-system is case-insensitive and then git gets confused.
But one just has to get along with that.

Reply:
As the class is actually new (I just renamed it in the PR for the sake of testing) it hopefully do not confuses windows to much :-)

Reply:
I appreciate the improved name.  It looks like a typo before.  😕